pkg/serializer/split/split.go

```diff
@@ -28,16 +28,18 @@ const (
 )
 
 var (
-	splitterExpvars    = expvar.NewMap("splitter")
-	splitterNotTooBig  = expvar.Int{}
-	splitterTooBig     = expvar.Int{}
-	splitterTotalLoops = expvar.Int{}
+	splitterExpvars      = expvar.NewMap("splitter")
+	splitterNotTooBig    = expvar.Int{}
+	splitterTooBig       = expvar.Int{}
+	splitterTotalLoops   = expvar.Int{}
+	splitterPayloadDrops = expvar.Int{}
 )
 
 func init() {
 	splitterExpvars.Set("NotTooBig", &splitterNotTooBig)
 	splitterExpvars.Set("TooBig", &splitterTooBig)
 	splitterExpvars.Set("TotalLoops", &splitterTotalLoops)
+	splitterExpvars.Set("Drops", &splitterPayloadDrops)
```

Comment:
Another PR of yours reminded me that we need to update https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/dist/conf.d/go_expvar.d/agent_stats.yaml.example since we introduce a new expvar.

Reply:
I'll add that